README.md

```diff
@@ -497,6 +497,8 @@ Now that you have created two tables, you'll see entries for them in the `pointc
 >     14ae4781464090c2f5285cbf5fc0ec51b81e858b46400ad7
 >     a3703dba5fc0ec51b81e858b46400ad7a3703dba5fc0e17a
 >     14ae4781464090c2f5285cbf5fc0e17a14ae47814640
+>
+> **PC_Envelope** is an alias to **PC_EnvelopeBinary**. But **PC_Envelope** is deprecated and will be removed in a future version (2.0) of the extension. **PC_EnvelopeAsBinary** is to be used instead.
```

Comment:
I would find this too confusing to the reader. So I'd just say that `PC_Envelope` is deprecated and `PC_EnvelopeAsBinary` should be used instead.